AnkiDroid/src/main/java/com/ichi2/anki/preferences/AppearanceSettingsFragment.kt

```diff
@@ -125,7 +125,8 @@ class AppearanceSettingsFragment : SettingsFragment() {
         requirePreference<SwitchPreferenceCompat>(R.string.show_estimates_preference).apply {
             launchCatchingTask { isChecked = withCol { config.get("estTimes") ?: true } }
             setOnPreferenceChangeListener { newETA ->
-                launchCatchingTask { withCol { config.set("estTimes", newETA) } }
+                val newETABool = newETA as? Boolean ?: false
+                launchCatchingTask { withCol { config.set("estTimes", newETABool) } }
             }
```

Comment:
Here's what I mean with returning false at the listener
```suggestion
            setOnPreferenceChangeListener { _, newETA ->
                val newETABool = newETA as? Boolean ?: return@setOnPreferenceChangeListener false
                launchCatchingTask { withCol { config.set("estTimes", newETABool) } }
                true
            }
```

Reply:
should i create new pull request with the requested changes?